Slingshot.Api/Helpers/Constants.cs

```diff
@@ -18,6 +13,19 @@ public class Repository
             public const string EmptySiteTemplateUrl = "http://deployredirector.azurewebsites.net/sitewithrepository.json";
             public const string GitCustomTemplateFolderUrlFormat = "https://raw.githubusercontent.com/{0}/{1}/{2}/";
             public const string GitHubApiRepoInfoFormat = "https://api.github.com/repos/{0}/{1}";
+            public const string BitbucketApiRepoInfoFormat = "https://api.bitbucket.org/2.0/repositories/{0}/{1}";
+            public const string BitBucketApiCommitsInfoFormat = "https://api.bitbucket.org/2.0/repositories/{0}/{1}/commits/{2}";
+
+            // BUG: "Missing default branch in v2 repository API when query public repo"
+            // https://bitbucket.org/site/master/issues/11732/missing-default-branch-in-v2-repository
+            //var defaultBranch = responseObj["default_branch"];
+            //if (defaultBranch == null)
+            //{
+            //    throw new ArgumentException("Could not discover default branch from repository");
+            //}
+            //_branch = defaultBranch.Value<string>();
```

Comment:
the comment is to remind us to use v2 API later. i will remove the commented code.